flappy_automation_code/scripts/flappy_automation_code_node.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Instead of editing the shebang, I would recommend using `catkin_install_python()`. That will rewrite the shebang for you to the correct python version in both the devel and install space.

```cmake
catkin_install_python(PROGRAMS scripts/flappy_automation_code_node.py
  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```

Here's some info about it in the `UsingPython3` tutorial for ROS 1 on the ROS wiki: http://wiki.ros.org/UsingPython3/SourceCodeChanges#Changing_shebangs